CONTRIBUTING.md

```diff
@@ -211,6 +211,14 @@ To reduce the effort required to review documentation-related changes,
 be viewed by clicking the `details` link on the
 `netlify/tracing-rs/deploy-preview check` on all pull requests.
 
+### Building Documentation
+
+If you want to build docs locally, you should do like this.
```

Comment:
It would be nice to explain briefly why this is necessary rather than just running `cargo doc`